### DISCUSSION TITLE
Is is possible to use Next.js for SSR with Spring(Java) without the need of setting up a separate express server?

### DISCUSSION THREAD

**Discussion by nisarhassan12** [vercel/next.js#12049] [Help]
Hi! :slightly_smiling_face: 

We are looking into using Next.js for server rendering the frontend of https://open-vsx.org (A vendor-neutral open-source alternative to the Visual Studio Marketplace). 

The backend for open-vsx is powered by Spring(Java). Is it possible to do SSR on runtime with Spring without the need of deploying a separate Node sever?

Thanks.

**Comment by elliotak47** [comment]
i'm looking for the same - i have created a web app using springboot as backend and next js as client end . but don't no how to build / package it ...
